### PR TITLE
Fixed login and sign up issues

### PR DESCRIPTION
**Pull request by DwinaTech**
Change isLoading to equal false if user is password is not match to stop spinner loading also reset login value to be true and sign up, reset password to equal false to make sign up  form redirect after user register successfully.

### CODE REVIEW

**Comment by kkarimi** [comment]
@DwinaTech is this necessary after @LouiaR 's PR?

**Comment by ibramoha** [comment]
LGTM

File: backend/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-DATABASE_URL='postgres://dos:dos@localhost:5432/dos'
+DATABASE_URL='postgres://dos:dos@192.168.99.100:5432/dos'
```
Comment (kkarimi) [inline backend/.env]: @DwinaTech we need to ignore this file somehow again and then you can keep your local one without needing to push

Comment (DwinaTech) [reply backend/.env]: Done

**Pull request merged**